src/main/resources/wa-task-permissions-wa-wacasetype.dmn

```diff
@@ -377,7 +377,7 @@ else
           <text>1</text>
         </outputEntry>
         <outputEntry id="LiteralExpression_1xaeelf">
-          <text>false</text>
```

Comment:
Unit test is failing. Corresponding unit test needs fixing

Reply:
fixed